runhouse/rns/function.py

```diff
@@ -55,9 +56,10 @@ def __init__(
         self.access = access or self.DEFAULT_ACCESS
         self.dryrun = dryrun
         self.resources = resources or {}
+
         super().__init__(name=name, dryrun=dryrun)
 
-        self = self.to(self.system, env=self.env)
```

Comment:
`.to` returns a new object, so I think in this case we do want to keep the `self = `?